user_journey/user_journey.model.lkml

```diff
@@ -16,6 +16,15 @@ explore: funnel_analysis {
               OR (${days_of_use.submission_date} = DATE_SUB(current_date, INTERVAL 2 DAY)
                   AND DATE_ADD(${funnel_analysis.submission_date}, INTERVAL 56 DAY) >  DATE_SUB(current_date, INTERVAL 2 DAY)));;
   }
+  join: client_properties {
+    relationship: one_to_one
+    type: left_outer
+    sql_where: client_properties.submission_date BETWEEN DATE_SUB(DATE({% date_start funnel_analysis.date %}), INTERVAL {% parameter client_properties.diff_days %} DAY) AND DATE_SUB(DATE({% date_end funnel_analysis.date %}), INTERVAL {% parameter client_properties.diff_days %} DAY);;
```

Comment:
Can you confirm that this results in the appropriate partition selection in BigQuery? Sometimes Looker uses timestamp types and BQ isn't able to push down the predicate.

Reply:
Yes, it does use the right partition. Initially, I was running into that issue but after explicitly casting everything to `DATE`s here it worked correctly.